src/input.rs

```diff
@@ -557,10 +557,30 @@ impl<'a, A: ActionContext + 'a> Processor<'a, A> {
         let mut args = launcher.args().to_vec();
         args.push(text);
 
-        match Command::new(launcher.program()).args(&args).spawn() {
-            Ok(_) => debug!("Launched: {} {:?}", launcher.program(), args),
-            Err(_) => warn!("Unable to launch: {} {:?}", launcher.program(), args),
-        }
+        #[cfg(not(windows))]
+        let _spawned = Command::new(launcher.program())
```

Comment:
The variable shouldn't be prefixed with `_`. This is usually used to prevent warnings about unused variables and thus could lead to issues when refactoring code in the future.